pkg/apis/core/v1beta1/federatedtypeconfig_types.go

```diff
@@ -219,7 +219,9 @@ func (f *FederatedTypeConfig) GetStatusType() *metav1.APIResource {
 }
 
 func (f *FederatedTypeConfig) GetStatusEnabled() bool {
-	return f.Spec.StatusCollection != nil && *f.Spec.StatusCollection == StatusCollectionEnabled
+	return f.Spec.StatusCollection != nil &&
+		*f.Spec.StatusCollection == StatusCollectionEnabled &&
+		f.Spec.FederatedType.Kind == "FederatedService"
```

Comment:
The name of the federated type is not necessarily  `FederatedService`.  Please check instead that name of the FTC is `services`.

Reply:
done